src/trace/context.spec.ts

```diff
@@ -1109,3 +1146,141 @@ describe("extractTraceContext", () => {
     `);
   });
 });
+
+describe("test_hexToBinary", () => {
```

Comment:
nit: Might be worth using a table driven test here to cut down on boilerplate: 
https://jestjs.io/docs/api#describeeachtablename-fn-timeout